### ISSUE TITLE
Suggestion: Ability to change spell ID

### ISSUE THREAD

**Issue by JyriaLycion** [stoneharry/WoW-Spell-Editor#162]
Just a minor suggestion:
I was wondering if its possible to add a button to swap spell ID's. So instead of cloning spell 1000 to 2000 and deleting 1000, u just change the spell ID 1000 to 2000.

**Comment by stoneharry** [comment]
Yes, but I would rather redesign that entire panel rather than add a new button. If you see the Spell Visual Kit, Effects, and Attachments editor it allows you to right click list items and get actions like copy/paste/delete. I want to implement something similar in the spell selection list such that you can in-line edit a lot of stuff. It's on my never-ending to-do list.
